### PR TITLE
update ipfs retrieval endpoints

### LINKED ISSUE #498: Update IPFS Fetching Endpoint
The endpoint used for **fetching** from IPFS should be updated to `https://scopelift.b-cdn.net/ipfs/<cid>/`. The trailing slash after the CID is *required* or the endpoint will 500.

This issue is a spinout from #486

### PR DESCRIPTION
**Pull request by mk1r**
resolves #498

### CODE REVIEW

**Review by mds1 — APPROVED** [review]

File: app/src/utils/data/ipfs.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { BigNumber } from 'src/utils/ethers';
 import { LocalForageData } from 'src/types';
 import { Ref } from 'vue';
 
-const retrievalEndpoint = 'https://ipfs.fleek.co/ipfs';
+const retrievalEndpoint = 'https://scopelift.b-cdn.net/ipfs/<cid>/';
```
Comment (apbendi) [inline app/src/utils/data/ipfs.ts:9]: The `/<cid>/` part was meant to be a placeholder for the content's CID.

**Review by apbendi — CHANGES_REQUESTED** [review]

File: app/src/utils/data/ipfs.ts (hunk 1 above)
Comment (gdixon) [inline app/src/utils/data/ipfs.ts:9]: I don't think we should be including `<cid>` in the endpoint:

```suggestion
const retrievalEndpoint = 'https://scopelift.b-cdn.net/ipfs';
```

(
[https://scopelift.b-cdn.net/ipfs/<cid\>/QmTQMgoxDRj8gfNn5Cvznt5CoEE6cz9MQeZrQsNb36BMdm ](https://scopelift.b-cdn.net/ipfs/<cid>/QmTQMgoxDRj8gfNn5Cvznt5CoEE6cz9MQeZrQsNb36BMdm 
)
vs 
https://scopelift.b-cdn.net/ipfs/QmTQMgoxDRj8gfNn5Cvznt5CoEE6cz9MQeZrQsNb36BMdm
)

@apbendi - I see in the issue you spoke about the trailing slash but it wasn't there previously - do we need it?

**Review by gdixon — CHANGES_REQUESTED** [review]

File: app/src/utils/data/ipfs.ts (hunk 1 above)
Comment (gdixon) [reply app/src/utils/data/ipfs.ts:9]: Lol, I just commented on this - I'll delete it now! 🚀

Comment (mds1) [reply app/src/utils/data/ipfs.ts:9]: Clearly I reviewed too quickly, whoops. Thanks all

**Comment by mk1r** [comment]
ah I see makes sense, I was wondering about that lol but then forgot to check

good catches

File: app/src/utils/data/ipfs.ts (hunk 1 above)
Comment (mk1r) [reply app/src/utils/data/ipfs.ts:9]: so, can it be `https://scopelift.b-cdn.net/ipfs`

Comment (gdixon) [reply app/src/utils/data/ipfs.ts:9]: I think that should work - the URL above resolves so I don't think we need that trailing slash

**Comment by mk1r** [comment]
updated, thanks team

**Review by gdixon — APPROVED** [review]

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**